prespective_rectification/utils.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+
+def convert_theta_to_two_points(x, y, theta):
+    x2 = x + 1000 * np.cos(theta)
+    y2 = y + 1000 * np.sin(theta)
+    x1 = x - 1000 * np.cos(theta)
+    y1 = y - 1000 * np.sin(theta)
+    line = [x1, y1, x2, y2]
+    return line
+
+
+def convert_theta_rho_to_two_points(rho, theta):
+    if rho < 0:
+        rho *= -1
+        theta -= np.pi
+    a = np.cos(theta)
+    b = np.sin(theta)
+    x0 = a * rho
+    y0 = b * rho
+    x1 = int(x0 + 1000 * (-b))
+    y1 = int(y0 + 1000 * a)
+    x2 = int(x0 - 1000 * (-b))
+    y2 = int(y0 - 1000 * a)
+    line = [x1, y1, x2, y2]
+    return line
+
+
+def get_intersection(polynomial_coeff_list, type_intersection="rho_theta"):
+    # Get intersections with lines
+    list_intersections = []
+    for i, _ in enumerate(polynomial_coeff_list):
+        for j, _ in enumerate(polynomial_coeff_list):
+            if i != j:
```

Comment:
Better to `continue` early:
```
if i == j:
   continue
try:
   ...
```
Saves you an indentation...

Reply:
cleaner yeah